docs/en/pid-eaa-data-model.rst

```diff
@@ -149,20 +156,24 @@ The ``record`` MUST have at least the following sub parameters:
     - **Description**
     - **Reference**
   * - **type** 
-    - identification of the national eID framework used by the User. For example ``eidas.it.cie`` means that the CIE id identification scheme is used by the User. 
+    - Identification of the trust framework used for obtaining the verified claims. For example, in case of PID, ``eidas.it.cie`` means that the CIE id identification scheme is used by the User. 
     - `[OID.IDA. Section 5.1.1.2] <https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html#section-5.1.1.2>`_
   * - **source**
     - JSON Object cointaining the follwoing mandatory claims:
 
-      - **organization_name**: Name of the Organization handling the eID used by the User
-      - **organization_id**: Identification code for the Organization. It MUST be set to the *IPA Code* of the Organization
+      - **organization_name**: Name of the Organization. In case of PID, it is the Organization handling the eID used by the User. For the (Q)EAA it is the Authentic Source.
+      - **organization_id**: Identification code for the Organization. For public Organization, it MUST be set to the *IPA Code*.
```

Comment:
I suggest to create a namespace here to uniquely identify the type of the value

Reply:
Ok, can you provide a proposal?

Reply:
done